common/display/src/main/scala/uk/ac/wellcome/display/models/v2/DisplayCollectionPath.scala

```diff
@@ -0,0 +1,44 @@
+package uk.ac.wellcome.display.models.v2
```

Comment:
Note that this file is renamed from `DisplayCollection.scala` but github hasnt picked up on that so all looks new,